.vscode/launch.json

```
@@ -0,0 +1,14 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Listen for Xdebug",
+      "type": "php",
+      "request": "launch",
+      "port": 9003,
+      "pathMappings": {
+        "/var/www/html/wp-content/plugins/remote-data-blocks": "${workspaceFolder}/"
```

```suggestion
        "/var/www/html/wp-content/plugins/remote-data-blocks": "${workspaceFolder}/",
        "/var/www/html/wp-content/plugins/example": "${workspaceFolder}/example"
```

This makes it work for debugging examples as well.